### DISCUSSION TITLE
Github Sponsors on iOS app?

### DISCUSSION THREAD

**Discussion by rockinrobstar** [community/community#40396] [Other Feature Feedback, Questions, & Ideas]
Is there a way to use Github Sponsors on the iOS app (preferably using say Apple Pay?)

**Comment by ossanna16** [comment]
Hi there @rockinrobstar and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
